packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -213,6 +213,16 @@ export interface GraphWidgetProps extends MetricWidgetProps {
    * @default TimeSeries
    */
   readonly view?: GraphWidgetView;
+
+  /**
+   * Whether to show the value from the entire time range. Only applicable for Bar and Pie charts.
+   *
+   * If false, values will be from the most recent period of your chosen time range;
+   * if true, shows the value from the entire time range.
+   *
+   * @default false
+   */
+  readonly setPeriodToTimeRange?: boolean;
```

Comment:
That's a weird name for a property, but I s'pose it's easiest/least ambiguous to be consistent with CloudWatch.

Reply:
I can't find this documented - https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html.

Have you tested this actually works?

Reply:
I obtained the property from actually building a dashboard and looking at the "source". I added the property to the integ test and verified it deployed. I also hacked this into our infrastructure pipeline via an override. Feel free to ship that PR first, and we'll see if it works as expected. :)